components/brave_wallet/browser/zcash/zcash_wallet_service.cc

```diff
@@ -330,9 +330,21 @@ void ZCashWalletService::CreateTransaction(const std::string& chain_id,
                                            const std::string& address_to,
                                            uint64_t amount,
                                            CreateTransactionCallback callback) {
+  std::string final_address = address_to;
+  if (IsUnifiedAddress(address_to)) {
+    auto transparent =
+        ExtractTransparentPart(address_to, chain_id == mojom::kZCashTestnet);
+    if (!transparent) {
+      std::move(callback).Run(base::unexpected("Wrong unified address format"));
```

Comment:
Shouldn't this text be in a grd so it can be properly translated? Also Wrong -> Invalid?